paien.yaml

```diff
@@ -384,7 +384,12 @@ packages:
       - quantum-espresso@6.2.0+mpi+scalapack
         ^fftw@3.3.7+mpi~openmp+fma simd=sse2,avx,avx2,avx512
 #       - namd fftw=mkl interface=python @2.12 ^charm backend=mpi ^python@2.7.14+tk
-#       - amber@16+mpi ^netcdf-fortran@4.4.4 ^netcdf@4.4.1.1+mpi ^hdf5@1.10.1+szip+mpi~cxx ^py-scipy@0.19.0 ^py-matplotlib@2.0.0 ^py-numpy+blas+lapack@1.12.1 ^python@2.7.13
+      - amber@16+mpi
+        ^netcdf-fortran@4.4.4 ^netcdf@4.6.1+mpi ^hdf5@1.10.1+szip+mpi+hl+fortran+cxx
+        ^py-scipy@1.1.0
+        ^py-numpy+blas+lapack@1.14.3
```

Comment:
This will pull a new numpy ? py-numpy in installed with `openmp threads=openmp` and this will go with openmp thread=none

Reply:
Will fix this once @batatasfritas 's modifications to Amber gets merged.